packages/teams-js/src/public/runtime.ts

```diff
@@ -121,19 +122,84 @@ export const teamsRuntimeConfig: IRuntime = {
       backStack: {},
       fullTrust: {},
     },
-    people: {},
     remoteCamera: {},
     sharing: {},
     teams: {
-      fullTrust: {
-        joinedTeams: {},
-      },
+      fullTrust: {},
     },
     teamsCore: {},
     video: {},
   },
 };
 
+// object of version constants
+export const versionConstants = {
+  '1.9.0': {
+    capabilities: { location: {} },
+    hostClientTypes: [
```

Comment:
this seems exactly the same as the one below. Can we extra it to only have the list once? Would that affect the bundle size?

Reply:
What does it mean to "extra it"?

Reply:
typo. Should have been 'extract'. As in have either a constant or a variable with the value and use it in two places.

Reply:
I've been tinkering with a couple ways to do this- I thought first that it would be awesome just to do `hostClientTypes: Object.values(HostClientType),` because I thought it was a clean and clear way to say "this works in all the host client types." And then I started to wonder if we would add any host client types in the future. The version constants would then automatically say they work in that host client type even if they aren't working there yet. Another option then is to declare a separate constant like you suggested, but I wasn't sure what to name it since the only thing that would separate it from any other host client types list would be the date/current version that I'm making it on, and that seems like an unfortunate way to name it. 

In the end though, this is suppose to serve an edge case for the few use cases for legacy hosts and by the time we might be adding new host client types it should hopefully be long in the past- so maybe it's a better choice to make it clean and clear to read. Would love to hear your thoughts

Reply:
Good catch on the `Object.values(HostClientType)` issue. I think I would define a constant and name it something like `v1HostClientTypes` to make it clear 1) why it's defined 2) what it represents.

Reply:
Done! Thanks :)